README.md

```diff
@@ -2,13 +2,13 @@
 
 [![Build Status](https://travis-ci.org/ayojs/ayo.svg?style=flat&branch=master)](https://travis-ci.org/ayojs/ayo?branch=master)
 
-_(Note: Ayo.js is forked from [Node.js][]. Currently, a lot of the documentation
+_(Note: Ayo.js is forked from [Node.js](https://nodejs.org). Currently, a lot of the documentation
 still points towards the Node.js repository.)_
 
-Ayo.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. Ayo.js
+Ayo.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. It
 uses an event-driven, non-blocking I/O model that makes it lightweight and
-efficient. The Ayo.js package ecosystem, [npm][], is the largest ecosystem of
```

Comment:
same comment here with the npm link - see line 96.

Reply:
It auto-links... something. I thought it would be more helpful if we actually inserted the wanted URL. But of course, it's no big deal.

Reply:
It links to the explicit URL defined on line 96, because it has the name "npm" - currently https://www.npmjs.com/. Your change adds the same URL, but instead of making it be a clean footnote, it inlines the long link directly in the prose.